tdrs-backend/tdpservice/reports/test/conftest.py

```diff
@@ -1,19 +1,17 @@
 """Report file fixtures."""
 
-import uuid
 import pytest
 
 @pytest.fixture
-def report_data():
+def report_data(user):
     """Return report creation data."""
     return {
-        "id": uuid.uuid4(),
```

Comment:
In the prior PR there was discussion that the report filename in S3 will be a UUID from the frontend, is that still the case? Just wondering why this line is being backed out

Reply:
that was not ultimately needed for testing because it is generated on its own.